### ISSUE TITLE
index.json url not referencing correct url

### ISSUE THREAD

**Issue by themestechnology** [slashdotdash/jekyll-lunr-js-search#107]
Hello I have noticed that when the Jekyll output is in a sub folder ie. "domain.com/myfolder/" then the {{URL}} only references the path without sub folder ie, "/folder2" rather than "/myfolder/folder2/"

Has anyone else experienced this issue and is there a way to set an absolute path ie. "domain.com/myfolder/folder2/index.html"

**Comment by cheshire137** [comment]
I'm seeing this too. Looks like https://github.com/slashdotdash/jekyll-lunr-js-search/issues/41 but using `{{baseurl}}` in my Mustache template didn't help.

**Comment by themestechnology** [comment]
Yes I am getting this too. I am using /:categories/:year/:month/:day/:title/

I noticed that there were 3 posts in one "day/" folder post 1 was created correctly and had a sub folder equal to the post title and an index.html in as it should but the other 2 posts were .html files each named as the post title rather than a folder being created
